src/errorLogger.js

```diff
@@ -0,0 +1,23 @@
+import Coralogix from 'coralogix-logger'
+
+const loggerConfig = new Coralogix.LoggerConfig({
+  applicationName: 'streetwise-stage',
+  privateKey: process.env.VUE_APP_FRONTEND_LOGGER_KEY,
+  subsystemName: 'frontend'
+})
+
+Coralogix.CoralogixLogger.configure(loggerConfig)
+const logger = new Coralogix.CoralogixLogger('FrontEnd')
+
+function logError (fileName = '', funcName = '', errorMessage = '') {
+  const error = new Coralogix.Log({
```

Comment:
When Coralogix is not set up, log errors simply to the window console.

Reply:
Hmm true. I will wrap this in try catch and default to just console logs.